sql/moz-fx-data-shared-prod/search_derived/search_revenue_levers_daily_v1/query.sql

```diff
@@ -96,7 +96,7 @@ mobile_dau_data AS (
     `moz-fx-data-shared-prod.telemetry.active_users_aggregates_device`
   WHERE
     submission_date = @submission_date
-    AND app_name IN ('Fenix', 'Firefox iOS', 'Focus Android', 'Focus Android')
+    AND app_name IN ('Fenix', 'Firefox iOS', 'Focus Android')
```

Comment:
`Focus Android` was doubled here.

Other than that this line stays unchanged because of https://github.com/mozilla/bigquery-etl/pull/3673/files#diff-636ec0757d00e0ec641af80fd2eb3bf50be69b9a2bcc840fdcae8e83489de429R7-R8